agent/service/hostapd.go

```diff
@@ -123,7 +124,17 @@ func hostapdConfigFile(radioConfig *ocstruct.OpenconfigOfficeAp_Radios_Radio_Con
 
 		// Add WLAN configuration.
 		wlanBridgeName := getBridgeName(int(*wlanConfig.VlanId))
-		hostapdWLANConfig := fmt.Sprintf(wlanConfigTemplate, wlanName, wlanBridgeName)
+
+		wlanStationIsolation := 0
+		if wlanConfig.StationIsolation != nil{
```

Comment:
The following may be more straightforward:

wlanStationIsolation := 0
if wlanConfig.StationIsolation != nil && *wlanConfig.StationIsolation {
  wlanStationIsolation = 1
}

Reply:
yes it is, updated.